src/emulator/modules/SceGxm/SceGxm.cpp

```diff
@@ -45,14 +45,7 @@ EXPORT(int, sceGxmBeginCommandList) {
 }
 
 EXPORT(int, sceGxmBeginScene, SceGxmContext *context, unsigned int flags, const SceGxmRenderTarget *renderTarget, const SceGxmValidRegion *validRegion, SceGxmSyncObject *vertexSyncObject, SceGxmSyncObject *fragmentSyncObject, const emu::SceGxmColorSurface *colorSurface, const emu::SceGxmDepthStencilSurface *depthStencil) {
-    assert(context != nullptr);
-    assert(flags == 0);
-    assert(renderTarget != nullptr);
-    assert(validRegion == nullptr);
-    assert(vertexSyncObject == nullptr);
-    assert(fragmentSyncObject != nullptr);
-    assert(colorSurface != nullptr);
-    assert(depthStencil != nullptr);
+    verify(HERE), context, flags == 0, renderTarget, !validRegion, !vertexSyncObject, fragmentSyncObject, colorSurface, depthStencil;
```

Comment:
I think we should avoid adding yet another way to trap/log errors.

I suspect most of the asserts can be replaced with:
- Proper error handling (which gets logged and returns a SCE error code),
- LOG_WARN_IF or similar
Or could just be deleted.

Reply:
Additionally, the comma operator just looks weird and non-idiomatic. People need to do a double-take to understand what's going on.

Reply:
Ok.